rllib/agents/slateq/slateq_tf_policy.py

```diff
@@ -204,7 +204,6 @@ def build_slateq_stats(policy: Policy, batch) -> Dict[str, TensorType]:
         "next_q_target_slate": policy._next_q_target_slate,
         "next_q_target_max": policy._next_q_target_max,
         "target_clicked": policy._target_clicked,
-        "td_error": policy._td_error,
```

Comment:
why not keep the stats here?

Reply:
We put them "back" in the `extra_grad_fetches` part. Can't wait to use your new Policy classes :D